docs/rewriter/examples/broadcast_matmul.py

```diff
@@ -0,0 +1,209 @@
+"""Onnx Pattern Rewriting with match condition parameter.
+
+This script shows how to define a rewriting rule based on patterns while
+utilizing the match condition parameter.
+
+First we write a dummy model with a several Reshape nodes and a Matmul node
+===================
+"""
+
+
+import math
+import numpy as np
+import torch
+import onnx
+import onnx.helper as oh
+import onnx.numpy_helper as onh
```

Comment:
```suggestion
import onnx.numpy_helper
```

avoid unconventional abbreviations